open_rarity/models/collection.py

```diff
@@ -47,13 +49,15 @@ class Collection:
         All trait names and values should be lowercased.
     name: A reference string only used for debugger log lines
 
+    We do not recommend resetting @tokens attribute after Collection initialization
```

Comment:
can we enforce it?

Reply:
We could do something like
```
class Person(object):
    def __init__(self, name, ssn):
        self.name = name
        self._ssn = ssn
    ssn = property(operator.attrgetter("_ssn"))
```
but then tokens wouldnt be an attribute but a property

Reply:
ok , not a blocker